addon/components/nrg-home-cards/home-card.js

```diff
@@ -22,9 +22,9 @@ export default class HomeCard extends Component {
   }
 
   @action click() {
-    const route = this.args.route;
-    const routeModel = this.args.routeModel;
-    const url = this.args.url;
+    const route = this.route;
+    const routeModel = this.routeModel;
+    const url = this.url;
```

Comment:
Same as above - the `args` hash should be needed since these values are passed in to the component.